.changeset/thin-crabs-care.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
> @tatianau @oksanashopify
> 
> https://screenshot.click/23-33-u4wzq-3p5jx.mp4
> 
> this look alright?


@knames  I think it's great! :raised_hands:  It solves it from the a11y perspective which is our ultimate goal behind this :pray:

I'm only curious about what the expected experience here is for the popover state UX-wise from the Polaris team perspective and whether it matters at all (cc @chloerice):
- Do we want to _make the dropdown open_ and return the focus to the specific action that opened the modal as in the [video above](https://screenshot.click/23-33-u4wzq-3p5jx.mp4)?
  - It's different from the existing production experience where the [dropdown closes](https://user-images.githubusercontent.com/15054990/227347609-3075f159-0a5e-4280-b954-0659440af17f.mp4
) when the modal is open, so the UX will change if we ship it
- Or do we expect to _keep the dropdown closed_ when the modal is open and return the focus to the `More action` dropdown CTA itself instead when closed?